_posts/blog/2016-09-13-code-reviews.md

```diff
@@ -0,0 +1,111 @@
+
+Even more, given as a command, even if correct, the advice won't be accepted wholeheartedly and the team morale is damaged.
+
+
+If the reviewer had invested just a little more time and watched the way they communicated: the author might have been very thankful and might have accepted all advice with a smile.
+
+
+The effort is well invested if one takes a break from time to time and makes sure the mindset is tuned, communication style is respectful and reflects the intention to help.
+
+
+## Don't Enforce Consensus
+
+Engineers often tend to forget that there is more than one valid opinion in a lot of cases. __Not everything is determined to false and true__.
+
+So if the reviewer has communicated their opinion appropriately and was not able to convince the author, the code review is the wrong forum to let discussion escalate and enforce consensus. If the code is robust and fulfills the requirements, let it pass the review and discuss the matter in an appropriate format. You may have tech meetings or other forums in which you can discuss the topic (ideally in general and __not__ by specific example) with a broader audience.
+
+Letting the topic lie for a week can also help. Often times you realize that consensus __isn't important at all__. That not only there are more valid opinions than yours. But also that your team maybe can afford to __let both opinions stand__.
+
+## Don't Point At Rules - Give Explanations
+
+Rules and processes kill __creativity__. You want to give as much freedom to everyone as possible without hurting someone else (that's part of the German constitution by the way [Article 2.1 _Every person shall have the right to free development ... insofar as he does not violate the rights of others ..._]).
+
+Developers construct processes and methods. They align on conventions. Sometimes other developers violate them; often even deliberately. How to handle those violations?
+
+Often when the reviewer changes from consultant to finger-pointing judge (and hangman) who points at the wiki in which he documented the rule three years ago (without ever discussing it properly); the tone gets rough. _Catch some breath, adjust your mindset..._.
+
+This behavior is bad because creative people have the habit to question rules if they are in their way. If the result is that the rule doesn't create value (but suppresses value-creation) the engineer, striving to achieve a goal, will try to navigate around it. In that case: __question the rule__, not the violator. In fact questioning and violating rules is a __healthy habit__.
+
+Feedback should be stated like this: _At some point we decided to approach things like this like that because we were convinced that's better because of reason X and Y. Don't you agree? Shall we re-evaluate?_
```

Comment:
How about putting this and the quote from Peter from below in a quote box, like Tomek is doing [here](https://upday.github.io/blog/subjects_neither_good_nor_evil/)